Configuration/PyReleaseValidation/scripts/runTheMatrix.py

```diff
@@ -303,6 +332,41 @@ def runSelected(opt):
                       default=None,
                       action='store')
 
+    parser.add_option('--RequiresGPU',
+                      help='if GPU is reuired or not: forbidden (default, CPU-only), optional, required',
+                      dest='RequiresGPU',
+                      default=0)
+
+    parser.add_option('--GPUMemory',
+                      help='to specify GPU memory. Default = 8000 MB (for RequiresGPU = required).',
+                      dest='GPUMemory',
+                      default='8000')
+
+    parser.add_option('--CUDACapabilities',
+                      help='to specify CUDA capabilities. Default = 7.5 (for RequiresGPU = required).',
+                      dest='CUDACapabilities',
+                      default='7.5')
```

Comment:
Why default only to `7.5`? I would think to default to all compute capabilities supported by the release. Which then raises two questions (that go somewhat beyond this PR though):

At this point we'd really need one source for the supported compute capabilities, because it is needed also in [`cudaIsEnabled`](https://github.com/cms-sw/cmssw/blob/master/HeterogeneousCore/CUDAServices/bin/cudaIsEnabled.cpp). An environment variable in `cuda-toolfile`? (not really my favorite but would be easy)

How to deal with different `SCRAM_ARCH`s supporting different sets of CUDA compute capabilities? E.g. our ARM build does not seem to support Pascal (6.x) while x86 and PPC do ([cuda-flags.file](https://github.com/cms-sw/cmsdist/blob/23d9ce99195f4fa32d265ad04facbf8bb581608a/cuda-flags.file)). 


Maybe also mention in the help that the value can be comma-separated.

Reply:
I will edit the help. However, to handle the default value, please suggest (or we can pick them from somewhere automatically).

Reply:
The `7.5` might be good-enough to get started, I suppose it depends mostly on what kind of hardware we are going to run on in the very near future. For a longer-term solution I opened an issue #33542.

Reply:
The current default should be `6.0,6.1,6.2,7.0,7.2,7.5`.